git-link.el

```diff
@@ -688,43 +736,7 @@ Defaults to \"origin\"."
            (list remote nil nil)
          (list remote (car region) (cadr region))))))
 
-  (let (filename branch commit handler remote-info (remote-url (git-link--remote-url remote)))
-    (if (null remote-url)
-        (message "Remote `%s' not found" remote)
-
-      (setq remote-info (git-link--parse-remote remote-url)
-            filename    (git-link--relative-filename)
-            branch      (git-link--branch)
-            commit      (git-link--commit)
-            handler     (git-link--handler git-link-remote-alist (car remote-info)))
-
-      (cond ((null filename)
-             (message "Can't figure out what to link to"))
-            ((null (car remote-info))
-             (message "Remote `%s' contains an unsupported URL" remote))
-            ((not (functionp handler))
-             (message "No handler found for %s" (car remote-info)))
-            ;; TODO: null ret val
-            (t
-             (let ((vc-revison (git-link--parse-vc-revision filename)))
-               (when vc-revison
-                 (setq filename (car vc-revison)
-                       commit   (cdr vc-revison)))
-
-               (git-link--new
-                (funcall handler
-                         (car remote-info)
-                         (cadr remote-info)
-                         filename
-                         (if (or (git-link--using-git-timemachine)
-                                 (git-link--using-magit-blob-mode)
-                                 vc-revison
-                                 git-link-use-commit)
-                             nil
-                           (url-hexify-string branch))
-                         commit
-                         start
-                         end))))))))
+  (git-link--new (git-link-url buffer-file-name remote start end)))
```

Comment:
Based on [my comment about `(message)`](https://github.com/sshaw/git-link/pull/69/files#r775096715) here will will now have catch any error and output it using `(message)`.